base/initdefs.jl

```diff
@@ -68,6 +68,23 @@ registries, packages, etc. installed and managed by system administrators.
 `DEPOT_PATH` is populated based on the [`JULIA_DEPOT_PATH`](@ref JULIA_DEPOT_PATH)
 environment variable if set.
 
+## DEPOT_PATH contents
+
+The directory in `DEPOT_PATH` contains a set of subdirectories, used for different purposes. Here is an overview
+of whats typically found there:
+
+* `clones`: Contains full clones of package repos. Maintained by `Pkg.jl` and used as a cache.
```

Comment:
So this gets populated, whenever I do a `] add SomePackage#some_branch_etc`? Anything else?

Reply:
Not entirely sure these days but close enough.